app/routes/translator/v1/health/index.js

```diff
@@ -10,7 +10,7 @@ const ctrl = require('../../../../controllers/status');
  */
 module.exports = function () {
     /** Platform of Trust health endpoint. */
-    router.get('', ctrl.healthCheck);
+    router.post('', ctrl.healthCheck);
```

Comment:
as far as I saw so far, a health endpoint is rather GET, not POST usually, right? not sure about this change :)